Sources/Solana/Api/getAccountInfo/getAccountInfo.swift

```diff
@@ -34,6 +45,13 @@ public extension Api {
         }
     }
 
+    /// Returns all information associated with the account of provided Publickey parsed
+    ///
+    /// - Parameters:
+    ///   - account: Publickey of account to query, as base-58 encoded string
+    ///   - decodedTo: Object from which the data value will be mapped. Must be BufferLayout implementation
+    ///   - allowUnfundedRecipient: If the account is empty it will not return a failure
+    ///   - onComplete: The result object of BufferInfo<T>. Where T is the decodedTo object. 
```

Comment:
```suggestion
    /// Returns all information associated with the account of provided `PublicKey` parsed
    ///
    /// - Parameters:
    ///   - account: `PublicKey` of account to query, as base-58 encoded string
    ///   - decodedTo: Object from which the data value will be mapped. Must be `BufferLayout` implementation
    ///   - allowUnfundedRecipient: If the account is empty it will not return a failure
    ///   - onComplete: The result object of `BufferInfo<T>`. Where `T` is the decodedTo object.
```